### DISCUSSION TITLE
Auto-Deploy command when setting up wings node puts http in the panel URL not https, causing "RecordNotFoundException" error when running wings.

### DISCUSSION THREAD

**Discussion by Boxersteavee** [pterodactyl/panel#5153] [General Community]
### Current Behavior

"Auto-Deploy" command outputs panel URL with http, not https. For example: `cd /etc/pterodactyl && sudo wings configure --panel-url http://panel.example.com --token [REDACTED] --node 5`

### Expected Behavior

"Auto-Deploy" command should give the URL in https if its in use. For Example: For example: `cd /etc/pterodactyl && sudo wings configure --panel-url https://panel.example.com --token [REDACTED] --node 5`

### Steps to Reproduce

1. Setup the panel, with the FQDN being HTTPS
2: Add a wings node, and select the auto-deploy command
3: Paste the auto-deploy command into the terminal, and look at the panel URL.

### Panel Version

1.11.3

### Wings Version

1.11.13

### Is there an existing issue for this?

- [X] I have searched the existing issues before opening this issue.
- [X] I have provided all relevant details, including the specific game and Docker images I am using if this issue is related to running a server.
- [X] I have checked in the Discord server and believe this is a bug with the software, and not a configuration issue with my specific system.

**Comment by danny6167** [comment]
This isn't a bug, The generated URL is based on the APP_URL you have specified in your .env file. Please correct your configuration.

**Comment by adit22-cyber** [comment]
cd /etc/pterodactyl && sudo wings configure --panel-url

**Comment by adit22-cyber** [comment]
http://panel.example.com --token [REDACTED] --node 5
